src/core_plugins/input_control_vis/public/components/vis/range_control.js

```diff
@@ -30,8 +30,18 @@ import {
 } from '@elastic/eui';
 
 const toState = (props) => {
+  let sliderValue = props.control.value;
+  if (!props.control.value) {
```

Comment:
should this be using `hasValue` as the check? Also looks like this function only uses `props.control` so no need to pass in the entire props.